Rls 0.6.0

TomAugspurger opened the pull request.

TomAugspurger committed:
RLS: 0.6.0

TomAugspurger committed:
MNT: Re-rendered with conda-smithy 3.1.5 and pinning 2018.05.22

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham commented:
Thanks @TomAugspurger. Couple suggestions:

1. Should switch to [using `pip`]( https://conda-forge.org/docs/meta.html#use-pip ) to build and install.
2. Should update how we [build against `numpy`]( https://conda-forge.org/docs/meta.html#building-against-numpy ).

TomAugspurger committed:
Update recipe

jakirkham committed:
Use `compiler` in `build` and drop `toolchain`

The `compiler` syntax is used to help us standardize compilers in
recipes (even across OSes). This replaces the now deprecated `toolchain`
and will be needed as we move to `defaults`' compilers.

jakirkham commented:
@conda-forge-admin, please re-render.

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 3.1.6 and pinning 2018.06.19

jakirkham committed:
Require `pip`

As `pip` is now used to `build` and `install` the package, add `pip` as
a requirement of `host` to ensure it is available to do this build and
install.

jakirkham committed:
Fix `numpy` pinning for Windows Python 3.6 builds

The oldest `numpy` on Windows for `python` 3.6 is 1.11. So bump the
pinning used during the build on that platform. On all other builds,
stick with `numpy` 1.9.

jakirkham commented:
@conda-forge-admin, please re-render.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I rerendered the feedstock and it seems to be already up-to-date.

jakirkham commented on recipe/meta.yaml:
```diff
@@ -13,23 +13,24 @@ source:
-    - toolchain
+    - pip
+    - numpy 1.9.*     # [not (win and py36)]
+    - numpy 1.11.*    # [win and py36]
```
Not sure how important `numpy` version `1.9` support is for us here. If not at all important, we can change this to just `numpy 1.11.*` (without selectors) or similar.

jakirkham commented:
Pushed a few changes to your PR to fix the build. Hope that is ok. Also please take a look and let me know if there are any questions.

TomAugspurger commented:
Thanks, I appreciate it.

I just did numpy 1.9 out of habit. Happy to bump it to 1.11.

________________________________
From: jakirkham <notifications@github.com>
Sent: Tuesday, June 19, 2018 10:18:39 AM
To: conda-forge/dask-ml-feedstock
Cc: Tom Augspurger; Mention
Subject: Re: [conda-forge/dask-ml-feedstock] Rls 0.6.0 (#4)


Pushed a few changes to your PR to fix the build. Hope that is ok. Also please take a look and let me know if there are any questions.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/conda-forge/dask-ml-feedstock/pull/4#issuecomment-398437521>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABQHIv8QCcXKznBLBNgJX2ygMMyVrX6iks5t-RZPgaJpZM4Urjd6>.

jakirkham committed:
Make `numpy` 1.11 the lower bound generally

Simplify things by making `numpy` version `1.11` the lower bound. Avoids
and OS/Python version specific constraints. Should also be easier to
maintain over the long run.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham commented:
Thanks :)

jakirkham commented:
All packages are now published. 🎉

TomAugspurger commented:
Thanks!

On Tue, Jun 19, 2018 at 4:31 PM, jakirkham <notifications@github.com> wrote:

> All packages are now published. 🎉
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/dask-ml-feedstock/pull/4#issuecomment-398553051>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABQHIpWI3mP-OiNfgPmc3jrbIRVOGiPGks5t-W25gaJpZM4Urjd6>
> .
>
